resource_estimation/RE_experiments.py

```diff
@@ -1,25 +1,41 @@
 from __future__ import annotations
 
-import os
-
-import qsharp
-from azure.quantum import Workspace
-from azure.quantum.chemistry import df_chemistry
-from azure.quantum.target.microsoft import MicrosoftEstimator, QECScheme, QubitParams
-
-resource_id = os.environ.get("AZURE_QUANTUM_RESOURCE_ID")
-location = os.environ.get("AZURE_QUANTUM_LOCATION")
-
-workspace = Workspace(resource_id=resource_id, location=location)
-estimator = MicrosoftEstimator(workspace)
-
-### Default qubit models
-
-params = estimator.make_params(num_items=6)
-
-# select Hamiltonian
-params.file_uris["fcidumpUri"] = "https://aka.ms/fcidump/XVIII-cas4-fb-64e-56o"
-
+from math import ceil
+
+from qsharp.estimator import EstimatorParams, EstimatorResult, LogicalCounts, QECScheme, QubitParams
+
+# For all experiments, we are using the logical resource counts as a starting
+# point.  These have been computed using the qsharp Python package (version
+# 1.6.0) for the https://aka.ms/fcidump/XVIII-cas4-fb-64e-56o Hamiltonian on
+# the sample
+# https://github.com/microsoft/qsharp/tree/main/samples/estimation/df-chemistry:
+#
+# ```
+# $ python chemistry.py -f https://aka.ms/fcidump/XVIII-cas4-fb-64e-56o
+# $ jq '.logicalCounts' < resource_estimate.json
+# ```
+
+logical_qubits = 0
+ccz_count = 0
+t_count = 0
+rotation_count = 0
+rotation_depth = 0
+measurement_count = 0
```

Comment:
Maybe I am overlooking something, but these lines/variables are not used anymore, correct?
```suggestion
```